Tensile/Common.py

```diff
@@ -1292,6 +1291,8 @@ def getArchitectureName(gfxName):
     "MinVgprNumber":                list(range(0,256)),
 
     "MaxVgprNumber":                list(range(0,257)),
+    # min K size to use GlobalSplitU algorithm 
+    "MinKForGSU":                   list(range(1,257))
```

Comment:
Why you changed this back to range?
Would you provide some explanation about your code change every time you submit new change?

Reply:
Nothing to prevent Tensile from choosing non power of 2 number to be MinKForGSU. If  you take a look at the added test. HGEMM performs better if MinKForGSU of value 35 is selected rather than 32.

Reply:
As I already said, MinKForGSU does not change any kernel code.
This is used just for reject condition.
You are comparing the result with the same kernel.

Reply:
> As I already said, MinKForGSU does not change any kernel code. This is used just for reject condition. You are comparing the result with the same kernel.

Even though MinKForGSU doesn't change kernel code. It, however, affect choosing GSU value. For example, GSU=16 is not a valid value if MinKForGSU=256, K=1024. We cannot limit MinKForGSU to power of 2 numbers, which may prevent us from choosing GSU=12 if it works better.

Reply:
> > As I already said, MinKForGSU does not change any kernel code. This is used just for reject condition. You are comparing the result with the same kernel.
> 
> Even though MinKForGSU doesn't change kernel code. It, however, affect choosing GSU value. For example, GSU=16 is not a valid value if MinKForGSU=256, K=1024. We cannot limit MinKForGSU to power of 2 numbers, which may prevent us from choosing GSU=12 if it works better.

We can enable any GSU values by using  power of 2 MinKForGSU  only.
We can set MinKForGSU=64 to use GSU=12 for K=1024.

Reply:
We would not like to have many duplicated kernels by using  slightly different MinKForGSU values.

Reply:
> We would not like to have many duplicated kernels by using slightly different MinKForGSU values.

I changed the definition of MinKForGSU to a list of [32,64,128,256].  It doesn't affect performance too much anyway.